### DISCUSSION TITLE
更新v2.1.3后，目录中汉字的红框高度不正常

### DISCUSSION THREAD

**Discussion by wohenbushuang** [sjtug/SJTUThesis#1053] [提问]
不知道是我环境的问题还是更新引入的（虽然完稿会把红框去掉…

更新后

![image](https://github.com/user-attachments/assets/0a8e5c23-6d35-4390-a6a9-c731c6710ca9)

更新前编译过的pdf

![image](https://github.com/user-attachments/assets/4cce598e-b18f-4d86-8a2e-53f5e0932c90)

**Comment by LogCreative** [comment]
应该也跟 fandol 字体有关，根据对应的平台换个其他字体比如 `cjk-font=ubuntu` 之类的应该就可以解决。
